.github/workflows/ci.yml

```diff
@@ -214,12 +214,13 @@ jobs:
       fail-fast: false
       matrix:
         platform: [macos-latest, ubuntu-latest] #windows-latest,
-        # python-version: [3.5, 3.6, 3.7, 3.8]
-        python-version: [3.6]
+        python-version: [3.6, 3.7, 3.8, 3.9]
     runs-on: ${{ matrix.platform }}
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Is that necessary?

Reply:
This PR uses [pypa/setuptools_scm](https://github.com/pypa/setuptools_scm) to 1) create the sdist and 2) automatically detect the package version from the latest tag (compliant with PEP440).

This means that the git clone must have all the history including the tags. However, inspecting the artifacts I see that `0.1.0.devX` is used. I typically use [another pattern](https://github.com/robotology/idyntree/blob/ab9336447ab0c6d6d9f9d0c3d3a19be31140aed3/.github/workflows/python.yml#L73-L74), I'll try to apply it also here and check if the versioning get fixed.

Reply:
Update in c2a9678, I will check the new artifacts if the version is fixed.